core/renderers/common/marker_svg.ts

```diff
@@ -214,10 +212,6 @@ export class MarkerSvg {
     }
   }
 
-  /**************************
-   * Display
-   **************************/
```

Comment:
Doe this (and similar below) no longer convey any useful information?  Personally I find dividers like this fairly helpful.